### DISCUSSION TITLE
nav links contains .html suffix

### DISCUSSION THREAD

**Discussion by bilal-fazlani** [squidfunk/mkdocs-material#2814] [Q&A] [answered]
Hi,
Version 7.1.8 & 7.1.9 introduces a different behaviour for urls / navigation links.

the urls end with `.html` 

With version 7.1.7 and earlier, urls were plain without `.html` prefix.

Does anyone know why that is? and what I can do to prevent `.html` in navigation links / urls?

version 7.1.8
![image](https://user-images.githubusercontent.com/3396271/124464952-71514200-ddb2-11eb-977d-313717bee23d.png)

version 7.1.7
![image](https://user-images.githubusercontent.com/3396271/124464973-757d5f80-ddb2-11eb-85d4-9861f3aeb047.png)

**Comment by squidfunk** [comment, marked as answer]
This is, very likely, not related to Material for MkDocs but to setting `use_directory_urls: false` in `mkdocs.yml`.

__Related__
- https://github.com/mkdocs/mkdocs/pull/2405
- https://github.com/mkdocs/mkdocs/issues/2189 (lengthy discussion)

This seems to be a new behavior of MkDocs 1.2.1.
